database/migrations/2017_03_13_212917_CreateSharedExpenseTable.php

```diff
@@ -18,6 +18,7 @@ public function up()
             $table->float('amount_owed');
             $table->string('secondary_username');
             $table->string('comments');
+            $table->timestamp('date_added');
```

Comment:
we should have `date_settled` right? @brynbeaudry i think you'd probably want that for graphs right?